### PR TITLE
[C-887] Host sourcemaps on s3, fixes cloudflare deploy

### PR DESCRIPTION
**Pull request by sliptype**
### Description

* Add ci to upload sourcemaps to `sourcemaps.audius.co` s3 bucket
* Remove sourcemaps from build deployed to cloudflare
* Update craco config to point at s3 bucket for sourcemaps
* Remove `@types/webpack` (included with webpack v5)
* Uses `cimg/python:3.7.12-node` image for cloudflare deploy steps because it has both node 14 and python

Was initially going to do the sourcemap move as a separate job but that required attaching and persisting the workspace which added unnecessary run time.

### Dragons

Right now all source maps are `sync`ed into the same directory in the bucket, this is never getting pruned so could get big/expensive. Can do another pass to create separate directories for the different builds, and overwrite existing files. But wanna merge this to unblock builds for now

### How Has This Been Tested?

Tested by deploying to staging and checking new sourcemaps work

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/craco.config.ts
Comment (piazzatron) [inline packages/web/craco.config.ts:64]: nit: maybe pull out into a const?

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sk-sourcemap-external

**Review by piazzatron — APPROVED** [review]
Glorious, thanks for handling this. My one thought is we should confirm that the sentry step still works at some point after this change, but not blocking merging this imo

**Pull request merged**
